src/templating/filesystem.ts

```diff
@@ -156,6 +167,17 @@ export async function writeFiles(
           title: 'Installing Dependencies',
           task: () => installDependencies(file.content, targetDir),
         };
+      } else if (file.type === 'README.md') {
+        const readmePath = path.join(readmesTargetDir, namespace + '.md');
```

Comment:
Since we added support for writing multiple templates to the same namespace, should this name include the template name?

Reply:
Also how will this play with `create-twilio-function` where the namespace is not existing

Reply:
This should all work now!